### PR TITLE
Player Movement

### PR DESCRIPTION
**Pull request by RobertKeanu**
Implemented the player movement including animations.

### CODE REVIEW

File: Assets/Scripts/PlayerMovement.cs [hunk 1]
```diff
@@ -1,18 +1,54 @@
...
+using System.Security.Cryptography;
+using System.Threading;
 using UnityEngine;
 
 public class PlayerMovement : MonoBehaviour
 {
+
+    public float speed;
+    private Rigidbody2D myRigidbody;
+    private Vector3 change;
```
Comment (widdrr) [inline Assets/Scripts/PlayerMovement.cs:14]: Why are we using a 3D vector for 2D movement?

File: Assets/Art/Result.png
Comment (widdrr) [inline Assets/Art/Result.png:1]: Could we rename this accordingly?

File: Assets/Scripts/PlayerMovement.cs (hunk 1 above)
Comment (widdrr) [reply Assets/Scripts/PlayerMovement.cs:14]: resolved privately

**Review by tiberiusss26 — APPROVED** [review]
lgtm

File: Assets/Scripts/PlayerMovement.cs [hunk 2]
```diff
@@ -1,18 +1,54 @@
...
+using System.Diagnostics;
+using System.Security.Cryptography;
+using System.Threading;
 using UnityEngine;
 
 public class PlayerMovement : MonoBehaviour
 {
+
+    public float speed;
+    private Rigidbody2D myRigidbody;
```
Comment (widdrr) [inline Assets/Scripts/PlayerMovement.cs:13]: Maybe rename this to something like playerRigidbody?

File: Assets/Scripts/PlayerMovement.cs [hunk 3]
```diff
@@ -1,18 +1,54 @@
...
+            animator.SetBool("moving", false);
+        }
+
+    }
+
+    void MoveCharacter()
+    {
+        myRigidbody.MovePosition(
+          transform.position + change * speed * Time.deltaTime
+            );
```
Comment (widdrr) [inline Assets/Scripts/PlayerMovement.cs:52]: Perhaps some basic comments documenting what each function and branch does would help.

File: Assets/Scripts/PlayerMovement.cs (hunk 2 above)
Comment (RobertKeanu) [reply Assets/Scripts/PlayerMovement.cs]: fixed

**Comment by Denj1X** [comment]
Good one

**Review by Denj1X — APPROVED** [review]
Seems godlike level.

**Review by Petrica81 — APPROVED** [review]

**Pull request merged**